### PR TITLE
more docs / structure

### PR DESCRIPTION
**Pull request by ThomasWaldmann**
Please do not change the changelog.md until this PR is merged.

It will move to docs/source/changes.rst after this PR, so it can be used on github as well as from within the documentation on readthedocs.io.

### CODE REVIEW

File: docs/source/changes.rst [hunk 1]
```diff
@@ -94,8 +135,13 @@
...
 * Fixed overflow bug in Minute-Count+
 
-### Version 0.2 2019-04-26 Juergen Boehringer
+V0.2 2019-04-26 jb
+------------------
+
 * added 1 Minute RS232 (USB) logging mode
 
-### Version 0.1 2019-03-25 Juergen Boehringer
+V0.1 2019-03-25 jb
```
Comment (ThomasWaldmann) [inline docs/source/changes.rst]: I needed to shorten that a bit, so it looks prettier when rendered in the left sidebar on the generated html site.

**Comment by rexfue** [comment]
OK

**Comment by ThomasWaldmann** [comment]
merging this now to test whether the github <-> readthedocs integration works...

**Comment by ThomasWaldmann** [comment]
https://multigeiger.readthedocs.io/en/latest/index.html it worked.

**Pull request merged**